src/explorer.ts

```diff
@@ -127,6 +128,19 @@ class KubernetesWorkloadFolder extends KubernetesFolder {
     }
 }
 
+class KubernetesConfigFolder extends KubernetesFolder {
+    constructor() {
+        super("config", "Configs");
```

Comment:
For the display name I suggest something like "Configuration" rather than "Configs".

Reply:
Nice. Fixed.